### DISCUSSION TITLE
Icon is rendered incorrectly in the header

### DISCUSSION THREAD

**Discussion by botkero** [squidfunk/mkdocs-material#6314] [Q&A] [answered]
Hello,
I would like to integrate icons into the sidebar, and don't quite know whether this should theoretically be an insider feature only. Some things work though, but I noticed a bug if you don't be just “have one page” but have a folder structure. The section header does not render the icon on the page correctly. It is difficult to explain. That's why I put in two images and a minimal configuration.

__When I'm on the top of the page__:
[on_top_of_the_page](https://github.com/squidfunk/mkdocs-material/assets/105428599/327ffabd-14d7-41cd-b134-a3b2e247d1e1)

__When I scroll a bit down__:
[scrolling_bit_down](https://github.com/squidfunk/mkdocs-material/assets/105428599/39dfeb4f-35f7-4d82-9e67-35d64e7fceb6)

Here the minimal reproduction:
[bug_mkdocs.zip](https://github.com/squidfunk/mkdocs-material/files/13306250/bug_mkdocs.zip)

**Comment by alexvoss** [comment]
Ok, so I could reproduce this using Firefox and 9.4.8+insiders.4.43.0, see below. This occurs when scrolling down on the index page for the "More" section (only, as far as I can see):

<img width="363" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/4134224/d244118d-4c2b-486d-8004-f1a066dac012">

Could you please do us the favour and have a look trough the discussions and issues to see if this has been reported before? If not, could you create an issue with a minimal reproduction - same as what you posted here but using the instructions on [creating a minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/). It is important to remove as many other settings as possible while still reproducing the issue. That rules out any more complex interactions between settings - or flags them up.

**Reply by botkero** [reply]
Yes, thank you for the confirmation. I'll try to remove all other settings that are not necessary. Could take a while.

**Reply by squidfunk** [reply, marked as answer]
Just for keeping track of things: it's an upstream issue, as already mentioned in #6315.

**Reply by botkero** [reply]
Sorry, I was looking for icons and couldn't find the upstream issue. If I had searched for emojis, I might have found it sooner.

**Reply by squidfunk** [reply]
No worries, I agree that this was quite a little harder to find.
